pkg/selinux/selinux.go

```diff
@@ -409,19 +409,21 @@ func GetLxcContexts() (processLabel string, fileLabel string) {
 	}
 
 	if processLabel == "" || fileLabel == "" {
-		return "", ""
+		return "", "", nil
```

Comment:
I assumed that in those cases (no SELinux, no lxc_contexts file and invalid content on lxc_contexts) `GetLxcContexts()` is a no-op